### PR TITLE
Change bug tracker from sourceforge to github inside documentation

### PR DESCRIPTION
**Pull request by frlan**
Pretty much this is just changing the link from sourceforge issue tracker to github issue tracker. Also it fixes a few other links to plugin pages.

### CODE REVIEW

File: geanylatex/doc/geanylatex.tex [hunk 1]
```diff
@@ -940,7 +940,7 @@ \section{Known issues}
 At time of the the documentation was created no issue were known.
 Since this is only a snapshot, you will find more recent information
 for all reported issues bug tracking system of SF at \\
```
Comment (b4n) [inline geanylatex/doc/geanylatex.tex]: reference to "SF"

File: geanypg/doc/geanypg.html [hunk 2]
```diff
@@ -60,12 +60,10 @@
...
-You can checkout the current source code from the Subversion repository
-at Sourceforge.net.<br><br>
-Get the code from:<br>
-<tt>svn checkout
-http://geany-plugins.svn.sourceforge.net/svnroot/geany-plugins/trunk/geany-plugins/geanypg
-</tt><br><br>
+You can checkout the current source code from the git repository
+at github.com.<br><br>
+Get the code from:<br><br>
+<code>git checkout  https://github.com/geany/geany-plugins.git</code><br><br>
```
Comment (b4n) [inline geanypg/doc/geanypg.html]: double space

File: git-changebar/README [hunk 3]
```diff
@@ -35,6 +35,7 @@ To navigate through the hunks of the current file, you need to configure the
 plugin's *Go to next hunk* and *Go to previous hunk* keybindings in Geany's
 preferences dialog.
 
+
```
Comment (b4n) [inline git-changebar/README]: herm, ok

File: multiterm/README [hunk 4]
```diff
@@ -75,4 +75,4 @@ Bug reports and feature requests
 --------------------------------
 
 To report a bug or ask for a new feature, please use the Geany-Plugins tracker
-on SourceForge: https://sourceforge.net/projects/geany-plugins/
+on github:  https://github.com/geany/geany-plugins/issues
```
Comment (b4n) [inline multiterm/README]: double space

**Comment by b4n** [comment]
Maybe use *GitHub* instead of *github* when it's used as a name?

**Comment by eht16** [comment]
> Maybe use *GitHub* instead of *github* when it's used as a name?

+1

**Comment by b4n** [comment]
LGTM

**Pull request merged**